libgraph/test/projection.cpp

```diff
@@ -23,15 +23,12 @@ static cll::opt<std::string> nodeTypes(
 static cll::opt<std::string> edgeTypes(
     cll::Positional, cll::desc("<edge types to project>"), cll::init(""));
 
-using ProjectedPropertyGraphView = katana::PropertyGraphViews::ProjectedGraph;
-
 struct TempNodeProp : public katana::PODProperty<uint64_t> {};
 using NodeData = std::tuple<TempNodeProp>;
 using EdgeData = std::tuple<>;
 
-using ProjectedGraphView = katana::TypedPropertyGraphView<
-    ProjectedPropertyGraphView, NodeData, EdgeData>;
-using GNode = typename ProjectedGraphView::Node;
+using ProjectedGraph = katana::TypedPropertyGraph<NodeData, EdgeData>;
```

Comment:
this doesn't make sense

Reply:
I am sorry, @uditagar, could you elaborate? The reason for this change is that, since a projection is now a property graph, I can build an instance of `TypedPropertyGraph` on it directly, I don't need `TypedPropertyGraphView` anymore. Was this what prompted your comment?

Reply:
then I guess you can just use `Graph` instead of `ProjectedGraph`